src/net_processing.cpp

```diff
@@ -5787,6 +5795,21 @@ bool PeerManagerImpl::SendMessages(CNode* pto)
         if (!vInv.empty())
             m_connman.PushMessage(pto, msgMaker.Make(NetMsgType::INV, vInv));
 
+        //
+        // Message: reconciliation request
+        //
+        {
+            if (!m_chainman.ActiveChainstate().IsInitialBlockDownload()) {
```

Comment:
In 30c39c1316a7b5c1914654f2a1487309b6e550ac: If we're in IBD, wouldn't we put that peer in the end of the queue with no need (I mean, without doing a rec request in fact)?